examples/plain/ios/upload-react-native-dsyms.sh

```diff
@@ -0,0 +1,2 @@
+#!/usr/bin/env bash
+bugsnag-dsym-upload example-dsyms.zip
```

Comment:
Rather than uploading an example dSYM zip it could upload the dsym generated for the example app

e.g.
```
bugsnag-dsym-upload build/Build/Products/Release-iphonesimulator/
```